101-vsts-cloudloadtest-rig/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "VSTS-CloudLoadTest-SelfProvisionedRig",
+  "description": "This template can be used to provision IaaS VMs to run load test using VSTS CLT service.",
+  "summary": "Yet to come",
```

Comment:
Needs to say something descriptive as it shows up here https://azure.microsoft.com/en-us/documentation/templates/